src/app/user/Balances/balanceUtils.ts

```diff
@@ -1,9 +1,11 @@
-import { BigNumberish, ethers } from 'ethers'
 import { GetAddressTokenResult, TokenBalance } from '@/app/user/types'
 import { tokenContracts } from '@/lib/contracts'
+import { formatUnits } from 'viem'
 
-export const formatBalanceToHuman = (balance: BigNumberish, decimal = 18) =>
-  ethers.formatUnits(balance, decimal)
+export const formatBalanceToHuman = (balance: number | bigint | string, decimal = 18, decimalToShow = 18) => {
```

Comment:
Since this function is used in multiple parts within the UI, I wouldn't change it, but create a new one